src/components/List/List.stories.tsx

```diff
@@ -89,10 +89,25 @@ const CheckTemplate = args => {
         );
   };
 
+  return <List {...args} selected={[selectedItem]} onChange={checkClick} />;
+};
+
+const CheckboxTemplate = args => {
+  const [selectedItem, setSelectedItem] = useState(args.selected || []);
```

Comment:
after you add the default value for the selected this `|| []` won't be necessary